Can't build the solution in visual studio. The "DiscoverGame" task was not found.

### Are you on the latest version of Nitrox?

Yes, I'm on the latest commit of the repo (only for experts).

### Which OS are you playing on?

Windows

### On which platform did you buy Subnautica?

Other

### Describe the issue.

Current solution do not depend from new package (Nitrox.Discovery.MSBuild). Solution can not be build from IDE.

### Describe the steps you took to encounter the bug.

Open the solution. Run Build Solution.

### Relevant log output

Nitrox\Directory.Build.targets(4,9): error MSB4036: The "DiscoverGame" task was not found.

--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -61,7 +61,7 @@
                     <PrivateAssets>all</PrivateAssets>
                     <IncludeAssets>build; native; contentfiles; analyzers</IncludeAssets>
                 </PackageReference>
-                <PackageReference Include="Nitrox.Discovery.MSBuild" Version="0.0.3">
+                <PackageReference Include="Nitrox.Discovery.MSBuild" Version="0.0.5">
                     <PrivateAssets>all</PrivateAssets>
                     <IncludeAssets>build; native; contentfiles; analyzers</IncludeAssets>
                 </PackageReference>

--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <Target Name="FindGameAndIncludeReferences" BeforeTargets="ResolveAssemblyReferences" Condition="'$(UnityModLibrary)' == 'true' or '$(TestLibrary)' == 'true'">
+    <Target Name="FindGameAndIncludeReferences" BeforeTargets="ResolveAssemblyReferences" Condition="'$(_NitroxDiscovery_TaskAssembly)' != '' and ('$(UnityModLibrary)' == 'true' or '$(TestLibrary)' == 'true')">
         <DiscoverGame GameName="Subnautica" IntermediateOutputPath="$(IntermediateOutputPath)">
             <Output TaskParameter="GamePath" PropertyName="GameDir" />
         </DiscoverGame>

